Description: set K3b as the default handler for iso files rather than ark
Author: Jonathan Thomas <echidnaman@kubuntu.org>

---
 src/k3b.desktop |    1 +
 1 file changed, 1 insertion(+)

--- a/src/k3b.desktop
+++ b/src/k3b.desktop
@@ -142,3 +142,4 @@ X-KDE-StartupNotify=true
 X-KDE-NativeMimeType=application/x-k3b;
 Categories=KDE;Qt;AudioVideo;DiscBurning;System;Filesystem;
 X-DBUS-ServiceType=Unique
+InitialPreference=5
